cups/ppd.c

```diff
@@ -3430,7 +3430,7 @@ ppd_update_filters(ppd_file_t     *ppd,	/* I - PPD file */
 		srctype[256],
 		dstsuper[16],		/* Destination MIME media type */
 		dsttype[256],
-		program[1024],		/* Command to run */
+		program[1024] = {0},		/* Command to run */
```

Comment:
The initialization list is used in other places in the code, so the usage is fine.
@rikylescak but all other code in the block are declarations, so I would move the definition of ```program``` below it next time.

I'll fix it by myself.